src/main/java/Helloworld.java

```diff
@@ -1,2 +1,5 @@
-package PACKAGE_NAME;public class Helloworld {
+public class Helloworld {
```

Comment:
```suggestion
public class HelloWorld {
```
in Java we use camelCase for names and classes start from upper case letter